app/hooks/handle_send_message.ts

```diff
@@ -128,20 +128,16 @@ export const useHandleSendMessage = ({
         let response: CreateResponse;
         if (schedulingInfo) {
             response = await createScheduledPost(serverUrl, scheduledPostFromPost(post, schedulingInfo, postPriority, postFiles));
-        } else {
-            response = await createPost(serverUrl, post, postFiles);
-        }
-
-        if (response.error && isFromDraftView) {
-            showSnackBar({
-                barType: SNACK_BAR_TYPE.CREATE_POST_ERROR,
-                customMessage: getErrorMessage(response.error),
-                type: MESSAGE_TYPE.ERROR,
-            });
-            return response;
-        }
-
-        if (isFromDraftView) {
+            if (response.error) {
+                showSnackBar({
+                    barType: SNACK_BAR_TYPE.SCHEDULED_POST_CREATION_ERROR,
+                    customMessage: getErrorMessage(response.error),
+                    type: MESSAGE_TYPE.ERROR,
+                });
+            } else {
+                clearDraft();
```

Comment:
This is also waiting for the response..

Reply:
This handles the loading state of the bottomsheet footer button that appears after long-pressing the send button to create a scheduled post.